lib/promisify.js

```diff
@@ -0,0 +1,23 @@
+function promisify(callbackFn) {
```

Comment:
I thought that we agreed on using `bluebird`? It's in a task description. Bluebird BTW already has the promisifyAll method: http://bluebirdjs.com/docs/api/promise.promisifyall.html
So all changes that basically was needed is to create RPCClient the same way as it used to be, and return Bluebird.promisifyAll(rpcClient);

Reply:
As discussed in slack.  This to be redone then with bluebird.